### PR TITLE
New Azure Terraform Rules

### PR DESCRIPTION
**Pull request by rezoan**
- Added PR-AZR-0079-TRF
- Added PR-AZR-0080-TRF
- Added PR-AZR-0081-TRF
- Added PR-AZR-0083-TRF
- Added PR-AZR-0084-TRF
- Added PR-AZR-0086-TRF
- Fixed PR-AZR-0138-ARM
- Fixed rules naming conflict.

### CODE REVIEW

File: azure/terraform/appservice.rego [hunk 1]
```diff
@@ -667,4 +667,361 @@ app_service_remote_debugging_disabled_metadata := {
...
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    not site_config.python_version
+}
+
+azure_issue["app_service_python_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    to_number(site_config.python_version) != 3.4
```
Comment (farchide) [inline azure/terraform/appservice.rego:875]: I suggest to have the python version as a variable
every time a new version coming out we can update the variable int he rego file to update the test case

File: azure/terraform/appservice.rego [hunk 2]
```diff
@@ -667,4 +667,361 @@ app_service_remote_debugging_disabled_metadata := {
...
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    not site_config.php_version
+}
+
+azure_issue["app_service_php_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    to_number(site_config.php_version) != 7.4
```
Comment (farchide) [inline azure/terraform/appservice.rego:816]: move the php version to a variable in the rego

**Review by farchide — CHANGES_REQUESTED** [review]

File: azure/terraform/appservice.rego (hunk 1 above)
Comment (rezoan) [reply azure/terraform/appservice.rego]: fixed

File: azure/terraform/appservice.rego (hunk 2 above)
Comment (rezoan) [reply azure/terraform/appservice.rego]: fixed.

**Review by farchide — APPROVED** [review]

**Pull request merged**